src/warnet/client.py

```diff
@@ -6,28 +6,29 @@
 import docker
 
 from warnet.utils import parse_raw_messages
-from warnet.warnet import Warnet
 
 logger = logging.getLogger("warnet.client")
 
-
-def get_bitcoin_debug_log(network: str, index: int) -> str:
-    warnet = Warnet.from_network(network)
-    tank = warnet.tanks[index]
-    subdir = "/" if tank.bitcoin_network == "main" else f"{tank.bitcoin_network}/"
-    data, stat = tank.container.get_archive(f"/home/bitcoin/.bitcoin/{subdir}debug.log")
-    out = ""
+def get_file_from_container(container, path):
+    data, stat = container.get_archive(path)
```

Comment:
I think we'll also want to add a container abstract base class method for `get_file` here so that we can keep it portable between docker and k8s